src/FNA3D_Driver_D3D11.c

```diff
@@ -5735,7 +6000,7 @@ static HRESULT D3D11_PLATFORM_ResizeSwapChain(
 		0,			/* get width from window */
 		0,			/* get height from window */
 		DXGI_FORMAT_UNKNOWN,	/* keep the old format */
-		renderer->supportsTearing ? 2048 : 0 /* See CreateSwapChain */
+		renderer->supportsTearing ? DXGI_SWAP_CHAIN_FLAG_ALLOW_TEARING : 0 /* See CreateSwapChain */
```

Comment:
See earlier enum comment,

```suggestion
		renderer->supportsTearing ? 2048 : 0 /* See CreateSwapChain */
```

Reply:
Also addressed in latest commit, same as the original comment.